### DISCUSSION TITLE
Is there any way to change the runtimeConfig attribute defined in nuxt.config.ts in the plugin in nuxt 3.6.5 version?

### DISCUSSION THREAD

**Discussion by tanglongyan** [nuxt/nuxt#27158] [Questions] [answered]
![image](https://github.com/nuxt/nuxt/assets/44485600/23cb677f-c753-489e-9c54-1db44622c0d6)
![image](https://github.com/nuxt/nuxt/assets/44485600/0748b974-f0fa-4d9e-869a-f48209eb8fc9)

**Comment by xjccc** [comment]
I'm not use this code . Is probably could change public property?
```
const config = useRuntimeConfig()
config.public.currentSite = {}
```

**Reply by tanglongyan** [reply]
This way doesn't work, I've already tried it

**Reply by xjccc** [reply]
![image](https://github.com/nuxt/nuxt/assets/16478776/f78b5718-5163-41fd-8025-8f8954846e4c)
only client writable and reactive

**Reply by TheAlexLichter** [reply, marked as answer]
This will at some point be possible with https://github.com/nuxt/nuxt/pull/27117/

**Reply by mukhriddin-dev** [reply]
**Boom**

**Comment by tanglongyan** [comment]
> This will at some point be possible with #27117
But my nuxt version is 3.6.5. Upgrading to the latest nuxt version is currently too expensive for me. Is there any other way to achieve this effect in the previous version?
